fuel/transformers/__init__.py

```diff
@@ -271,12 +395,17 @@ class Cache(Transformer):
         refilled when needed through the :meth:`get_data` method.
 
     """
-    def __init__(self, data_stream, iteration_scheme):
+    def __init__(self, data_stream, iteration_scheme, **kwargs):
```

Comment:
No mention of axis labels here... I think it can be forced in the call to super().

Also, not sure how produces_examples should be handled, as I have no idea how this is used. Does it only emit example streams?